walkeeper/src/timeline.rs

```diff
@@ -515,3 +591,79 @@ impl Storage for FileStorage {
         Ok(())
     }
 }
+
+#[cfg(test)]
+mod test {
+    use super::FileStorage;
+    use crate::{
+        safekeeper::{SafeKeeperState, Storage},
+        timeline::{CreateControlFile, SharedState, CONTROL_FILE_NAME},
+        SafeKeeperConf,
+    };
+    use anyhow::Result;
+    use std::fs;
+    use zenith_utils::{lsn::Lsn, zid::ZTimelineId};
+
+    fn stub_conf() -> SafeKeeperConf {
+        let workdir = tempfile::tempdir().unwrap().into_path();
+        SafeKeeperConf {
+            workdir,
+            ..Default::default()
+        }
+    }
+
+    fn file_storage(
```

Comment:
`load_create_control_file`? It's basically a wrapper around `load_from_control_file`, and it's used both to get the `FileStorage` and load `SafeKeeperState` for checking.